src/MovieCard/MovieCard.js

```diff
@@ -52,38 +48,31 @@ class MovieCard extends Component {
         console.log(e);
       }
   }
-
   addsFavorite = async (event) => {
     try {
-      const movieIdToUpdate = event.target.id;
-      await postFavorite(movieIdToUpdate, this.props.userID)
+      const { id } = event.target;
+      await postFavorite(id, this.props.userID)
       this.props.getAllFavorites();
```

Comment:
we can just destructor id out of the object here, then it automatically becomes that variable for us to use on line 54